CONTRIBUTING.md

```diff
@@ -92,17 +92,3 @@ Amplify UI team publishes to NPM on every Tues (mandatory) and Thurs (optional).
 1. Changesets get accumulated into [Version Packages PR](https://github.com/aws-amplify/amplify-ui/pulls/app%2Fgithub-actions).
 1. Maintainer will run tests, approve, and merge the PR.
 1. Previous step will trigger [`publish-latest`](./.github/workflows/publish-next.yml) action that will build and publish packages to NPM.
-
-## Updating Icons
```

Comment:
Can you delete "Updating Icons" in table of contents too?

Reply:
Good catch! I may need to make sure if we want to delete the "Updating Icons" section first 😂

Reply:
Thanks for commenting. PR updated 😁